first_app/hello.py

```diff
@@ -37,7 +38,7 @@ def register_user_api():
     second_name = data["second_name"]
     password = data["password"]
     
-    user = User('phone_number', 'first_name', 'second_name', 'password')
+    user = User("phone_number", "first_name", "second_name", "password")
```

Comment:
Should be User(phone_number, first_name, second_name, password). Why you use strings here?

Reply:
Yes, I changed mistake already